lib/Scvmm/miq_winrm.rb

```diff
@@ -0,0 +1,53 @@
+
+require 'winrm'
+
+class MiqWinRM
+  attr_reader :uri, :username, :password, :hostname, :port, :connection
+
+  def initialize
+    port    = 5985
+    require 'uri'
+  end
+  
+  def build_uri
+    uri          = URI::HTTP.build(:port => @port, :path => "/wsman")
+    uri.hostname = @hostname
+    uri.to_s
```

Comment:
@jerryk55 
I like the MiqWinRM class. As it stands all this code is a duplication of much of EmsMicrosoft. I imagine we will want to update EmsMicrosoft to leverage this class too.